docs/components/beacon/index.html

```diff
@@ -100,4 +99,67 @@ <h2 id="beacon-with-inline-styles">Beacon with Inline Styles</h2>
       </footer>
     </div>
   </section>
+
+  <section>
+    <header>
+      <h2 id="beacon-with-default-styles">Beacon with Color Override</h2>
```

Comment:
Please change id to - `beacon-with-color-override`.

Reply:
Changes have been done, Please review